client/docs/client.md

```diff
@@ -0,0 +1 @@
+:::src.featureform.client
```

Comment:
what is this?

Reply:
I believe this is a routing mechanism for [`mkdocs`](https://www.mkdocs.org/), though, I've never actually looked into this. This file was autogenerated by `mkdocs`.